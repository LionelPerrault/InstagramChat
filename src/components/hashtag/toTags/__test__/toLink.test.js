import React from 'react'
import { create } from 'react-test-renderer'
import ToLink from '../toLink'

describe('ToLink Component', () => {
  it('should match snapshot', () => {
    const tree = create(
      <ToLink url="/hashtag/travel" label="A label" />
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
