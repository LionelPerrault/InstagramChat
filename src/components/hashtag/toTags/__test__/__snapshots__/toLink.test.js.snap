// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToLink Component should match snapshot 1`] = `
<Link
  className="hashtag"
  to="/hashtag/travel"
>
  A label
</Link>
`;
